src/data.js

```diff
@@ -9,6 +9,16 @@ var items = [
     discord: "https://discord.gg/ZURP4Mx4Vc",
     logo: "https://osu.gatari.pw/static/images/glyphs/glyph-white.svg"
   },
+  {
+    name: "tsuki!",
+    link: "https://tski.moe",
+    owner: "jupiter & salcido",
+    location: "AU",
+    onlineAPI: "https://api.tski.moe/get_online_count",
```

Comment:
```suggestion
    onlineAPI: "https://api.tski.moe/get_player_count",
```
it should be `player`